ingestion-beam/src/test/java/com/mozilla/telemetry/integration/BigQueryIntegrationTest.java

```diff
@@ -89,16 +90,45 @@ public void canWriteToBigQuery() throws Exception {
     String input = Resources.getResource("testdata/json-payload.ndjson").getPath();
     String output = String.format("%s:%s", projectId, tableSpec);
 
-    PipelineResult result = Sink.run(new String[] { "--inputFileFormat=text", "--inputType=file",
-        "--input=" + input, "--outputFileFormat=text", "--outputType=bigquery",
-        "--output=" + output, "--errorOutputType=stderr" });
+    PipelineResult result = Sink
+        .run(new String[] { "--inputFileFormat=text", "--inputType=file", "--input=" + input,
+            "--outputType=bigquery", "--output=" + output, "--errorOutputType=stderr" });
```

Comment:
Unrelated change: there was a spurious `"--outputFileFormat=text"` that I removed here.